bluesky/plan_stubs.py

```diff
@@ -892,6 +892,28 @@ def caching_repeater(n, plan):
         yield from (m for m in lst_plan)
 
 
+def count_step(detectors, step, pos_cache):
+    """Inner loop of a count scan
```

Comment:
```suggestion
    """Inner loop of a count scan.
```

Reply:
I remember @danielballan was mentioning this in some previous reviews, that it should be a complete sentence ending with the dot.